### DISCUSSION TITLE
Date Picker Form Component help

### DISCUSSION THREAD

**Discussion by adamsanclemente** [Budibase/budibase#4176] [Help] [answered]
I'm trying to use the date pick to choose a time for an event. I'm trying to store the chosen time as a datetime, however when I try to choose a field when using the date picker it says it's an invalid type or something. What's the proper way to go about doing this?

**Comment by mjashanks** [comment, marked as answer]
Hi @PixelParallax  - What database are you using to store the data?

**Reply by adamsanclemente** [reply]
MySQL

**Reply by mjashanks** [reply]
so, `date`, `datetime` or `timestamp` columns shoud be recognised as budibase "Date" types & usable by the date picker.

**Reply by adamsanclemente** [reply]
That's weird, it isn't.

**Reply by mjashanks** [reply]
If you click on your table, then edit the column - it should tell you what type that it Budibase thinks it is:
![image](https://user-images.githubusercontent.com/3524181/150826122-3eb74345-54a7-4e1b-8a26-04fb99542ebb.png)

![image](https://user-images.githubusercontent.com/3524181/150826233-712eae99-9a20-4ba5-961b-9001b2b0d943.png)

What does this screen tell you? What type is your column?

**Reply by adamsanclemente** [reply]
Yea it says text, I've updated it in heidisql, it should be datetime

**Reply by mjashanks** [reply]
Do you need to Fetch Tables again to refresh it in Budibase?

**Reply by adamsanclemente** [reply]
got it, thanks. Had to refresh
